### DISCUSSION TITLE
Does it support regular a1111 prompt format?

### DISCUSSION THREAD

**Discussion by Dachiko** [MoonRide303/Fooocus-MRE#139] [Q&A]
Like attention modifiers (small:0.5), random selection like {1-3$$__my/Place__|__my/event__|__my/Style__} and such?

**Comment by barepixels** [comment]
I know for sure it supports this  (small:0.5)

**Comment by lucas-0liveira** [comment]
As attention modifiers with a lower weight (small:0.5), random selection is made among options like {my/Place|my/event|my/Style} and similar ones.

**Reply by Dachiko** [reply]
Thanks! It would really help having a proper documentation on prompting techniques...

**Reply by lucas-0liveira** [reply]
Hi, can you mark your question as answered, Thanks ^^

**Reply by barepixels** [reply]
Can you give an example of {my/Place|my/event|my/Style}  in actual prompt?

**Reply by Dachiko** [reply]
Those were meant to be wildcards ("__ wildcards __") but here two underscores __ makes text bold. So my/Place is a wildcards folder "my" with Place.txt in there. But it seems Fooocus doesn't support subfolders.
